### ISSUE TITLE
JSONDecodeError

### ISSUE THREAD

**Issue by Emma-LM** [UMNFuN/FUNGuild#27]
Hi!

I am trying to run FUNGuild locally from the command line. I have followed your instructions. I have installed the Anaconda Python version 3.6.6; I have updated to the most recent version of pip (2.22.0); I have installed the requests package; I have saved Guilds_v1.1.py into the same folder as my otu file; and I have chaged the directory to this file. When I try and run the script, I get this error message:

S:\FUNGuild>python Guilds_v1.1.py -otu Test.txt -db fungi
FunGuild v1.0 Beta
Connecting with FUNGuild database ...
Traceback (most recent call last):
  File "Guilds_v1.1.py", line 128, in <module>
    current_record = json.loads(record)
  File "C:\Program Files\Python36\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Python36\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Python36\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 10 (char 9)

I don't think it is a problem with the otu file - as I have tried making just a very small one - exactly as specified in your instructions (plus the error does not mention problem with that?). 

Your help would be much appreciated.
Thanks,
Emma.

**Comment by ZeweiSong** [comment]
Can you attach your.otu table?

Zewei

获取 Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by Emma-LM** [comment]
OFFICIAL

Hi Zewei,

Thanks for getting back to me. I think I have worked it out. It wasn’t the OUT table – as when I downloaded your example and tried that it didn’t work. It was issues with our firewall.

I have had issues downloading and updating packages in R and python before – ICT don’t like to give us access to download anything. But these errors came up with something including access denied in the message so it was easy to work out what was wrong. I even had an issue just trying to look at you example file?!

But once I was given access via a different proxy to access the example file – it suddenly started working, and then suddenly stopped – I remembered that the alternative proxy connection times out after a while, and low and behold, when I switched it on again, it all worked fine.

I was just confused by the error message as it didn’t seem to be anything related to issues with the OTU table – or anything about access. And I couldn’t find any posts relating to this error and FUNGuild.

I have since tried my own OTU data and it seems to be working fine.

Anyway, hope that made sense – at the very least it might be worth noting if you see any issues like this in the future it may be due to access/firewall issues.

Hope that makes sense!
Emma.
